src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -92,6 +96,16 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
     forwardedRef.current?.close()
   }
 
+  const onPressRefresh = () => {
```

Comment:
```suggestion
  const onPressRefreshNextPage = () => {
```
actually i wonder if you want to consider combining this with `onPressTryAgain`? they're doing the same thing and could probably just take a `getNextPage` as input. we could also consider having the same analytics event with a property for `getNextPage` (since it's the same user action, to retry the fetch?)

Reply:
yeah, good callout, combined them into a single fn that shares an analytics event.